core/src/avm2/globals/flash/text/ime/IIMEClient.as

```diff
@@ -0,0 +1,25 @@
+package flash.text.ime {
+    import flash.geom.Rectangle;
+
+    public interface IIMEClient {
```

Comment:
Since we now early-lookup interfaces, some SWFs (at least #15485) break without this.